custom_components/teslafi/coordinator.py

```diff
@@ -123,7 +123,13 @@ async def _refresh(self) -> TeslaFiVehicle:
             )
         else:
             self._override_next_refresh = None
-
+            
+        # Refresh TeslaFi command counts
+        command_counts = await self._client.request_counts()
+        LOGGER.debug("TeslaFi command counts: %s", command_counts)
+        if command_counts is not None and "commands" in command_counts:
+            self._vehicle.update_non_empty(command_counts)
```

Comment:
Sanity check? It appears that we don't need this, because the standard polling request now includes the command counters:
```
"...,"tesla_request_counter":{"commands":8,"wakes":15,"data_source":"cache","rate_limit":{"reset_in_seconds":1754}}}
```

So we can just access it with `self._vehicle["tesla_request_counter"]["commands"]` and `"wakes"`. We already do that automatically, but only in response to [execute_command()](https://github.com/jhansche/ha-teslafi/blob/main/custom_components/teslafi/coordinator.py#L55): 
```python
        response = await self._client.command(cmd, **kwargs)
        self._vehicle.update_non_empty(response.get("tesla_request_counter", {}))
        self.async_set_updated_data(self._vehicle)
```
So I think we can simplify this significantly, without making an extra request:
```suggestion
        # Refresh TeslaFi command counts
        if (counts := current.get("tesla_request_counter", {}):
            self._vehicle.update_non_empty(counts)
```

This was what I was hoping/waiting for, after your last PR for this

Reply:
The "sanity check" comment was more me asking you to double check what I was seeing 🙂 to make sure I wasn't missing something.